src/composables/useRaycaster/index.ts

```diff
@@ -9,14 +9,13 @@ import type { DomEvent, TresCamera, TresEvent } from 'src/types'
 import type { TresContext } from '../useTresContextProvider'
 
 export const useRaycaster = (
-  objects: Ref<Object3D[]>,
+  objectWithEvents: Ref<Object3D[]>,
```

Comment:
Can we please change this to `objectsWithEvents`? (The original was plural and the type is an array, so many `objects`.)

Thanks! And sorry for the late change request here.

Reply:
Thank you for your prompt response